src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -462,12 +462,11 @@ public void incomingPacketInfo(
         if (logger.isTraceEnabled())
         {
             logger.trace(diagnosticContext
-                .makeTimeSeriesPoint("incoming_packet", nowMs)
+                .makeTimeSeriesPoint("in_pkt", nowMs)
                 .addKey("rbe", hashCode())
-                .addField("arrival_time_ms", arrivalTimeMs)
-                .addField("send_time_24bits", sendTime24bits)
-                .addField("send_time_ms", sendTimeMs)
-                .addField("payload_size", payloadSize)
+                .addField("recv", arrivalTimeMs)
+                .addField("send", sendTimeMs)
```

Comment:
`incoming_packet` -> `in_pkt` didn't seem too bad, but some of these shorter names add a lot of ambiguity.

Reply:
I agree, maybe I could add some clarifying suffixes, like '_ms' in the send/recv fields?

Reply:
`send_ts_ms`?